agent/internal/agent.go

```diff
@@ -84,6 +85,11 @@ func (a *Agent) start() {
 		return
 	}
 
+	// Create the command manager to process backend commands
+	commandMng := NewCommandManager(a, a.logger)
+	// Process commands that may have been received on login.
+	commandResults := commandMng.Do(appLoginRes.Commands)
```

Comment:
I belive this is `commands` with a lowercase C

Reply:
Are you talking about the json attribute names? If yes, then these names are defined by what is called "field tags" like [here](https://github.com/sqreen/go-agent/blob/master/agent/backend/api/api.pb.go#L275-L276). They are used by the generic json marshaler to generate the expected field names.

Side note: in Go, everything starting with a capital letter is made public, ie. exposed API of the package for others.

Reply:
LGTM